api/src/test/java/org/jmisb/api/klv/KlvParserTest.java

```diff
@@ -11,22 +11,18 @@
 import java.util.List;
 import java.util.SortedMap;
 import java.util.TreeMap;
-import org.apache.log4j.AppenderSkeleton;
-import org.apache.log4j.Logger;
-import org.apache.log4j.spi.LoggingEvent;
-import org.mockito.ArgumentCaptor;
-import org.mockito.Captor;
-import org.mockito.Mock;
-import org.mockito.Mockito;
-import org.mockito.MockitoAnnotations;
+import org.testng.annotations.AfterMethod;
 import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeTest;
+import uk.org.lidalia.slf4jext.Level;
+import uk.org.lidalia.slf4jtest.LoggingEvent;
+import uk.org.lidalia.slf4jtest.TestLogger;
+import uk.org.lidalia.slf4jtest.TestLoggerFactory;
 
 public class KlvParserTest
```

Comment:
Can this also use LoggerChecks?

Reply:
This one is a bit more complex. The tests aren't checking the primary logger, so it would require a big extension of the LoggerChecks code. If there was another case of that, I would have centralised the test code, but it didn't seem worth it for just this class.